viewer/packages/rendering/src/glsl/base/determineMatrixOverride.glsl

```diff
@@ -1,4 +1,5 @@
-#pragma glslify: unpackVec4ToFloat = require('../color/unpackVec4ToFloat.glsl')
+precision highp float;

```

Comment:
Is this really necessary? It should at least not be in a import?

Reply:
Make sense to add it into import as we have update to `glslify-import`, will update it